### DISCUSSION TITLE
Assets by Website

### DISCUSSION THREAD

**Discussion by sinnathurai** [craftcms/cms#13069] [Q&A] [answered]
Hello everyone, 
We would like to use one Azure Blob Storage Assets Volume per website. Is there already a solution for this? or a workaround?

Craft CMS: 4.4.5
Azure Blob Storage Plugin: 2.0.0

**Comment by brandonkelly** [comment, marked as answer]
Volumes are site-agnostic in Craft, but you can create an Assets field that uses the volume, which only appears on a single site. You can do that by adding a Site condition to the Assets field within field layout designers that incorporate it.
